.github/workflows/playwright.yml

```diff
@@ -0,0 +1,28 @@
+name: Playwright Tests
+on: pull_request
+jobs:
+ test:
+   timeout-minutes: 60
+   runs-on: ubuntu-latest
+   steps:
+   - uses: actions/checkout@v4
+   - uses: actions/setup-node@v4
+     with:
+       node-version-file: '.nvmrc'
+       cache: 'npm'
+       cache-dependency-path: 'package-lock.json'
+   - name: Install dependencies
+     run: npm ci
+
+   - name: Install Playwright Browsers
+     run: npx playwright install --with-deps
+
+   - name: Run Playwright tests
+     run: npx playwright test
+
+   - uses: actions/upload-artifact@v4
+     if: ${{ !cancelled() }}
```

Comment:
nit: it's up to the team to decide what to do here, but sometimes it's better to only store the playwright report if the tests fail. right now it looks like the report would be uploaded and stored as long as the job isn't cancelled.